src/app/components/hris/employees/employee-profile/accordions/accordion-administrative-documents/accordion-administrative-documents.component.ts

```diff
@@ -35,7 +35,7 @@ export class AccordionAdministrativeDocumentsComponent {
   selectedFile !: File;
   roles: string[] = [];
   isLoadingUpload: boolean = false;
-  allowedTypes = ['application/pdf'];
+  allowedTypes = ['application/pdf', 'image/png', 'image/jpeg'];
```

Comment:
Should we not also include image/jpg if that is a thing. JPEG and JPG are two different file types even though they are files used for images

Reply:
I removed the jpeg as the design only specifies jpg not jpeg. Small oversight